src/Datadog.Trace/Tagging/CosmosDbTags.cs

```diff
@@ -0,0 +1,28 @@
+using System.Linq;
+using Datadog.Trace.Configuration;
+using Datadog.Trace.ExtensionMethods;
+
+namespace Datadog.Trace.Tagging
+{
+    internal class CosmosDbTags : InstrumentationTags
+    {
+        protected static readonly IProperty<string>[] CosmosDbTagsProperties =
+            InstrumentationTagsProperties.Concat(
+                new ReadOnlyProperty<CosmosDbTags, string>(Trace.Tags.InstrumentationName, t => t.InstrumentationName),
+                new Property<CosmosDbTags, string>(Trace.Tags.CosmosDbContainer, t => t.ContainerId, (t, v) => t.ContainerId = v),
+                new Property<CosmosDbTags, string>(Trace.Tags.CosmosDbDatabase, t => t.DatabaseId, (t, v) => t.DatabaseId = v),
```

Comment:
Should we reuse `db.name` instead of adding the new tag name `cosmosdb.database`?
(but then what do we do with `cosmosdb.container`?)

Reply:
I've used `db.name`, I guess `cosmosdb.container` will be okay on it's own.